hoot-core-test/src/test/cpp/hoot/core/conflate/poi-polygon/PoiPolygonMatchTest.cpp

```diff
@@ -825,13 +825,15 @@ class PoiPolygonMatchTest : public HootTestFixture
       RecursiveElementRemover(w1->getElementId()).apply(map);
     }
 
+    // shop=mall has now been added to the building category, so whereas previously it couldn't
+    // be considered a building without also having building=yes, now it can be considered a
+    // building with the tag alone.
     {
```

Comment:
This test now yields a review instead of a miss, since shop=mall is in the building category...a good thing.